ConsoleApp1/Program.cs

```diff
@@ -4,6 +4,111 @@ internal class Program
 	{
 		static void Main(string[] args)
 		{
+			var initialVotes = new List<Vote>()
+			{
+				new Vote("v1"){
+					Questions = new List<Question>()
+					{
+						new Question("q1"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						}
+					}
+				},
+				new Vote("v2"){
+					Questions = new List<Question>()
+					{
+						new Question("q1"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						},
+						new Question("q2"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						}
+					}
+				},
+				new Vote("v3"){
+					Questions = new List<Question>()
+					{
+						new Question("q1"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						},
+						new Question("q2"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						},
+						new Question("q2"){
+							Answers = new List<Answer>()
+							{
+								new Answer("ans1"),
+								new Answer("ans2"),
+								new Answer("ans3"),
+							}
+						}
+					}
+				},
+			};
+
+
+			var voteSystem = new VoteSystem(null, null);
+
+			while (true)
+			{
+				DrawMenu();
+				var key = Console.ReadLine();
+				if (!new[] { "1", "2", "3", "5" }.Contains(key?.Trim())) { Console.WriteLine("Error. Try again. press 5 to Exit"); continue; }
+				switch (key)
+				{
+					case "1":
```

Comment:
Move all key-operation to the dictionary and reuse it here instead of switch

Reply:
there are no swith now =)))